avocado/core/safeloader.py

```diff
@@ -62,18 +62,33 @@ def iter_classes(self):
 
             # Looking for a 'import avocado'
             elif isinstance(statement, ast.Import):
-                for name in statement.names:
-                    if name.name == 'avocado':
-                        if name.asname is not None:
-                            self.mod_imports.add(name.asname)
-                        else:
-                            self.mod_imports.add(name.name)
+                imp_name = statement_import_as(statement).get('avocado', None)
+                if imp_name is not None:
+                    self.mod_imports.add(imp_name)
 
             # Looking for a 'class Anything(anything):'
             elif isinstance(statement, ast.ClassDef):
                 yield statement
 
 
+def statement_import_as(statement):
+    """
+    Returns a tuple of imported module names whether using aliases or not
```

Comment:
It actually returns a dictionary. It needs to `s/tuple/dictionary/` here.

Reply:
True, I was playing with a change here and left it like that.

Reply:
Fixing the docstring.